README.md

```diff
@@ -2,3 +2,7 @@ jscience
 ========
 
 A stripped down fork of JScience's unit library (version 4.3.1) with various bug fixes.
+
+* 4.3.2: Fixed an error with division order of operations.
+* 4.3.3: Removed currency exchange support (all currencies on own dimension).
+* 4.3.4: Added new dimensions to previously dimensionless concepts (radian, ppm, dB, and a few others). 
```

Comment:
I don't think it matters, but would these last two changes really be patch level numbers in semver terms? Seems like we significantly changed behavior.

Reply:
Agreed. Bumped to a major patch, so 5.0.0. Technically I don't think this qualifies as major, but the one before did.